lang/semgrep-grammars/src/semgrep-elixir/grammar.js

```diff
@@ -33,12 +33,28 @@ module.exports = grammar(base_grammar, {
     // Ellipsis
     // No need for extensions to _expressions for ellipsis because
     // Elixir already uses "..." as valid identifiers
-      
+    //
+    // However, we do need to override the "pair" rule which is used
+    // for keyword parameters, arguments, and map items. Otherwise,
+    // ellipsis won't work in places where Elixir expects
+    // keyword/pairs, such as
+    //   foo(some_arg: 0, ...)
+    //   %{some_item: 0, ...}
+    // Also note that now there is ambiguity whether foo(...) is an
+    // identity or pair, we set the pair rule to have a lower
+    // precedence
+    pair: ($, previous) => {
+      return prec(-1, choice(
+        previous,
+        '...',
```

Comment:
Typically, we will define the nonterminal `$.semgrep_ellipsis` to be `"..."` and use that, rather than inlining, because it will tend to produce more descriptive typed CSTs. 

Not a big deal here though, since this is the single use case in the file, so please don't go to the trouble of revising it, rereleasing `semgrep-elixir`, and having to update your other PR!

Reply:
Better to follow Brandon's advice though. You can look at the other semgrep-xxx/grammar.js for inspiration, but it helps to be consistent in our different grammar extensions.